easybuild/easyconfigs/s/scikit-learn/scikit-learn-1.0.2-foss-2021b.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
```

Comment:
@mustafabar Please change this to a bundle of `scikit-learn` + the `sklearn` metapackage, like we did in #15613